src/Datadog.Trace/Span.cs

```diff
@@ -17,6 +17,7 @@ namespace Datadog.Trace
     /// </summary>
     public class Span : IDisposable, ISpan
     {
+        internal const long Pow2e53 = 9007199254740992;
```

Comment:
nit: Could you add a comment or rename the variable to make it more clear that we're limited by the golang float64 type size?

Reply:
ok, sure